### PR TITLE
Save image, marker, title, user to event. save event to marker

### PR DESCRIPTION
**Pull request by caro5**
@sukhada

Added image and title to the events list view.
Marker shows up in the mapview when a new event is added from the timelineview

### CODE REVIEW

File: app/src/main/java/turtler/voyageur/fragments/TripMapFragment.java [hunk 1]
```diff
@@ -375,7 +375,7 @@ public void onClick(View view) {
                 final Event event = new Event();
```
Comment (sukhada) [inline app/src/main/java/turtler/voyageur/fragments/TripMapFragment.java:375]: i think you can remove `Trip t`?

File: app/src/main/java/turtler/voyageur/fragments/CreateEventFragment.java [hunk 2]
```diff
@@ -211,43 +229,47 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
...
-                ivPreview.setImageBitmap(selectedImage);
+                setFragmentUIWithEventProps();
             } else {
                 Toast.makeText(self, "No picture chosen!", Toast.LENGTH_SHORT).show();
             }
         }
     }
 
+    public void setFragmentUIWithEventProps() {
+        ivPreview.setImageBitmap(selectedImageBitmap);
```
Comment (sukhada) [inline app/src/main/java/turtler/voyageur/fragments/CreateEventFragment.java]: did you add the `ivPreview` back to the XML?

File: app/src/main/java/turtler/voyageur/fragments/TripMapFragment.java (hunk 1 above)
Comment (caro5) [reply app/src/main/java/turtler/voyageur/fragments/TripMapFragment.java:375]: we use it though in line 383

Comment (sukhada) [reply app/src/main/java/turtler/voyageur/fragments/TripMapFragment.java:375]: oops my bad!

**Pull request merged**